satellite/repair/checker/checkerstats.go

```diff
@@ -44,6 +46,13 @@ func (collector *statsCollector) collectAggregates() {
 	}
 }
 
+// combineCollectors fills stats of original statsCollector with stats of addition collector.
+func (collector *statsCollector) combineCollectors(addition *statsCollector) {
+	for scheme, stats := range addition.stats {
+		collector.stats[scheme] = stats
```

Comment:
Shouldn't stats be aggregated here instead of replaced?

Reply:
since combining *IntVal's is a bit complicated I moved collectAggregates() got Join and removed in Finish for now